admin_run.go

```diff
@@ -116,7 +116,7 @@ func (s *adminRuns) ForceCancel(ctx context.Context, runID string, options Admin
 
 // Check that the field RunStatus has a valid string value
 func (o AdminRunsListOptions) valid() error {
```

Comment:
I think the most important thing here is consistency and I'm fine leaving it as is. Since `valid()` doesn't mutate values or ever really accept a complex struct (except _maybe_ in `workspace.go`) I think a value receiver makes more sense.

Here's some nice reading: https://github.com/golang/go/wiki/CodeReviewComments#receiver-type